src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file Redesigned Metadata sidebar component
+ * @author Box
+ */
+import * as React from 'react';
+import flow from 'lodash/flow';
+import { FormattedMessage } from 'react-intl';
+import { withAPIContext } from '../common/api-context';
+import { withErrorBoundary } from '../common/error-boundary';
+import { withLogger } from '../common/logger';
+import { ORIGIN_METADATA_SIDEBAR_REDESIGN } from '../../constants';
+import messages from '../common/messages';
+
+import './MetadataSidebarRedesign.scss';
+
+function MetadataSidebarRedesign() {
+    return (
+        <div className="bcs-metadata-redesign">
```

Comment:
new classnames should follow SUIT naming:
`bcs-MetadataSidebarRedesign`

Reply:
Got it, renamed